### PR TITLE
fix: set max message size for grpc client

### PR DESCRIPTION
**Pull request by dakimura**
default message size limit is 4MB ([ref](https://github.com/tensorflow/serving/issues/1382))
but our usage might exceed the limit

### CODE REVIEW

File: pymarketstore/grpc_client.py [hunk 1]
```diff
@@ -23,7 +23,12 @@ class GRPCClient(object):
 
     def __init__(self, endpoint: str = 'localhost:5995'):
         self.endpoint = endpoint
-        self.channel = grpc.insecure_channel(endpoint)
+        # set max message sizes
+        options = [
+            ('grpc.max_send_message_length', 1 << 31 - 1),
+            ('grpc.max_receive_message_length', 1 << 31 - 1),
```
Comment (briancappello) [inline pymarketstore/grpc_client.py:29]: Exactly 1GB, cool... curious why not `1 << 30`? (or did you mean `(1 << 31) - 1` for 2GB minus 1byte?)

Comment (briancappello) [reply pymarketstore/grpc_client.py:29]: Also TIL, at least in Python, using shift is barely faster than `1 * 1024**3`, which might be more readable/obvious for non-low-level devs?
```
%timeit 1 << 30
4.46 ns ± 0.0184 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

%timeit 2**30
4.47 ns ± 0.0148 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

%timeit 1 * 1024**3
4.49 ns ± 0.0172 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)
```

**Comment by briancappello** [comment]
It's surprising to me that this is the client's responsibility to configure... hypothetically, let's say you're running `marketstore` on a server with only 4GB of RAM, and I (the client) configure the max to be 8GB and send you an 8GB request.. did I just crash your server? (I don't know if that's how it actually works.) But regardless, the link in your description makes it sound like both the client and the server need to configure these values (the [server-side default is 4MB as well](https://github.com/grpc/grpc-go/blob/master/server.go#L57))

File: pymarketstore/grpc_client.py (hunk 1 above)
Comment (dakimura) [reply pymarketstore/grpc_client.py:29]: sorry, I just copy-pasted the value from `math.MaxInt32` in marketstore code, but I agree that `1* 1024**3` should be much more readable. Let me change it...

**Comment by dakimura** [comment]
> It's surprising to me that this is the client's responsibility to configure... hypothetically, let's say you're running `marketstore` on a server with only 4GB of RAM, and I (the client) configure the max to be 8GB and send you an 8GB request.. did I just crash your server? (I don't know if that's how it actually works.) But regardless, the link in your description makes it sound like both the client and the server need to configure these values (the [server-side default is 4MB as well](https://github.com/grpc/grpc-go/blob/master/server.go#L57))

hmm I don't know how it actually works either... but it should make more sense that server-side returns a bad-request error or something to the client when the message is too big.
Let's set the value 1GB for now and set the limit values to the server too (Thank you for [your fix](https://github.com/alpacahq/marketstore/pull/364)!!

**Comment by ifokeev** [comment]
@dakimura how is it possible to pass message limit variables? No config or ENV options here

**Pull request merged**